spark-jobs/src/test/scala/za/co/absa/enceladus/conformance/streaming/HyperConformanceIntegrationSuite.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.conformance.streaming
+
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.catalyst.expressions.GenericRowWithSchema
+import org.scalatest.FunSuite
+import za.co.absa.enceladus.conformance.interpreter.fixtures.{NestedStructsFixture, StreamingFixture}
+
+class HyperConformanceIntegrationSuite extends FunSuite with StreamingFixture with NestedStructsFixture {
+
+  test("Test with catalyst workaround") {
+    val frame: DataFrame = testHyperConformance(standardizedDf,
+      "result",
+      nestedStructsDS)
+    frame.show()
```

Comment:
Don't use show in tests

Reply:
I mostly thought that show would make following what happens at the runtime easier

Reply:
I understand, but in an ideal world, you just look at the bottom where it says that everything passed. In case of an issue, every one of us can write this line by themselves and re-run